addons/udes_stock_picking_batch/__manifest__.py

```diff
@@ -16,6 +16,7 @@
         "udes_get_info",
         "udes_stock_refactoring",
         "udes_stock",
+        "package_hierarchy",
```

Comment:
Is this a missing dependency?

Reply:
I have no idea how this got here. I will double-check and remove if not needed.

Reply:
Not needed, not sure how it got there.